project-base/storefront/components/Blocks/SimpleNavigation/SimpleNavigationListItem.tsx

```diff
@@ -21,11 +21,28 @@ export const SimpleNavigationListItem: FC<SimpleNavigationListItemProps> = ({
     const itemImage = 'mainImage' in listedItem ? listedItem.mainImage : null;
     const href = getStringWithoutTrailingSlash(listedItem.slug) + '/';
 
+    const getLinkType = (type: ListedItemPropTypeTypename | undefined): PageType => {
+        switch (type) {
+            case 'ArticleSite':
+                return 'article';
+            case 'BlogArticle':
+                return 'blogArticle';
+            case 'Brand':
+                return 'brand';
+            case 'Category':
+                return 'category';
+            default:
+                throw new Error('Link type ' + type + ' could not be resolved.');
+        }
+    };
```

Comment:
Let's put this into some helper file. It doesn't have to be inside this component.